### ISSUE TITLE
Restore wrapper (and install?) generation rules (conditionally?)

### ISSUE THREAD

**Issue by alpmestan** [snowleopard/hadrian#539]
With #531 we're getting relocatable GHC builds and we therefore won't need any sort of wrapper scripts like `inplace/bin/ghc-stage2` (when building with the make build system) in most situations (we won't need them on Linux, OS X and Windows).

Some OSes will probably still need/want them so it would therefore be nice if we could eventually revive the wrapper generation rules, and have them fire only when building a GHC for an OS that's not one of the 3 aforementionned ones.

The question of the install rules (which we get rid of in #531 as well) also is in the air. With #531 installation is as easy as shipping `<build root>/stage1/{bin, lib, a few other files and dirs}`". We can still consider offering a rule/command to this effect.

**Comment by snowleopard** [comment]
@alpmestan Thank you! Can we list all files that need to be included in an installation build rule, if we decide to add it?

**Comment by alpmestan** [comment]
@snowleopard Hmm, I don't have that list handy, but the current code for Wrappers.hs (see [here](https://github.com/snowleopard/hadrian/blob/7a5c384dc9cb8ae9d41c5203074d1e3a79bcca67/src/Rules/Install.hs#L18) for a time-proof link) has some comments that describe the things to be installed and even shows the logic that we could steal and just adapt to the changes we do in #531. Is that a good enough answer? Or should I try and find/come up with an actual list?

**Comment by snowleopard** [comment]
@alpmestan Yes, that's a good start. I just want to make this issue self-contained so that it could potentially be tackled by a newcomer.
